apps/ios/GuideDogs.xcodeproj/project.pbxproj

```diff
@@ -6273,7 +6371,7 @@
 				SDKROOT = iphoneos;
 				SWIFT_COMPILATION_MODE = wholemodule;
 				SWIFT_OPTIMIZATION_LEVEL = "-O";
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 5.0;
```

Comment:
Is the swift version upgrade incidental, required or a good practice? Would it impact anything else? Perhaps @benjamintios would know (I don't, sorry).

Reply:
Swift version 5 seems well-established, hence resolving.